src/app/core/store/collections/collections.effects.ts

```diff
@@ -114,8 +114,8 @@ export class CollectionsEffects {
 
   @Effect()
   public delete$: Observable<Action> = this.actions$.ofType<CollectionsAction.Delete>(CollectionsActionType.DELETE).pipe(
-    switchMap(action => this.collectionService.removeCollection(action.payload.collectionCode)),
-    map(collectionCode => new CollectionsAction.DeleteSuccess({collectionCode: collectionCode})),
+    switchMap(action => this.collectionService.removeCollection(action.payload.collectionId)),
+    map(collectionId => new CollectionsAction.DeleteSuccess({collectionId: collectionId})),
```

Comment:
There is a shortcut for this. Instead of `{collectionId: collectionId}` you can write just `{collectionId}`.

You do not have to change it. I just wanted to let you know about the new ES6 syntax. ;)

Reply:
Yes I know about that, thanks for finding that.